tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -188,6 +188,8 @@ def loadbalancer_setup(
 
 @pytest.mark.e2e
 @pytest.mark.tier1
+@pytest.mark.pit_server
```

Comment:
Apologies, I missed this one in the first review. This one needs to go as well since it spins up capsule(s).

Note: it would be great to note all the capsule test cases somewhere so we can go back to it and mark them with `pit_server` once the missing AAP workflow is in place.

Reply:
Updated the PR.